shared/templates/template_OVAL_accounts_password

```diff
@@ -1,5 +1,11 @@
+{{% if rule_id -%}}
+{{% set oval_id = rule_id -%}}
+{{% else -%}}
+{{% set oval_id = "accounts_password_pam_" + VARIABLE -%}}
+{{% endif -%}}
+
 <def-group>
-  <definition class="compliance" id="accounts_password_pam_{{{ VARIABLE }}}" version="3">
+  <definition class="compliance" id="{{{ oval_id }}}" version="3">
```

Comment:
@matejak 

I think that @jhrozek has problem with file_permissions template and not with this one right now, but anyway it's a perfect start because I think we will fix all the templates we have.

Reply:
Yes, I would like to port all templates, but I have started with one and then just apply the same technique to the rest.